pinakes/main/approval/serializers.py

```diff
@@ -17,12 +18,30 @@
 from pinakes.main.approval import validations
 
 
+class NotificationSerializer(serializers.ModelSerializer):
+    """Notification method that define what settings are expected"""
+
+    class Meta:
+        model = Notification
+        fields = ("id", "name", "setting_schema")
+
+
 class TemplateSerializer(serializers.ModelSerializer):
     """The template to categorize workflows"""
 
     class Meta:
         model = Template
-        fields = ("id", "title", "description", "created_at", "updated_at")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "created_at",
+            "updated_at",
+            "process_method",
+            "process_setting",
+            "signal_method",
```

Comment:
@bzwei How do the signals help here, are they for mails in the queue to use new settings.